oci-unit-tests/cassandra_test.sh

```diff
@@ -12,7 +12,11 @@
 #  setUp() - run before each test
 #  tearDown() - run after each test
 
-readonly CQLSH_DOCKER_IMAGE="docker.io/cassandra:latest"
+readonly UPSTREAM_DOCKER_IMAGE="docker.io/cassandra:latest"
+readonly CQLSH_DOCKER_IMAGE="cassandra-cqlsh:test"
+# We build the cqlsh image locally because upstream does not ship a s390x image
+# See https://github.com/canonical/server-test-scripts/pull/130
+readonly USE_UPSTREAM_IMAGE=false
```

Comment:
My only comment would be to use less generic variables names, just in case we decide in the future the `USE_UPSTREAM_IMAGE` should be a global variable or some such.  WDYT?